executor/runtime/types/sidecars.go

```diff
@@ -215,9 +215,9 @@ func shouldStartLogViewer(cfg *config.Config, c Container) bool {
 }
 
 func shouldStartTitusStorage(cfg *config.Config, c Container) bool {
-	// Currently titus-storage only supports EBS and /mnt-shared storage
+	// Currently titus-storage sets up NFS, EBS, and /mnt-shared storage
 	// which is currently only available on multi-container workloads
-	return c.EBSInfo().VolumeID != "" || len(c.Pod().Spec.Containers) > 1
+	return len(c.NFSMounts()) > 0 || c.EBSInfo().VolumeID != "" || len(c.Pod().Spec.Containers) > 1
```

Comment:
This is great. Maybe on the next pass we can remove the whole `c.NFSMounts()` idea entirely!

Reply:
Do you mean removing nfs mounts or just from the pod container object?

Reply:
Removing the `NFSMounts()` function itself. 
Don't do it in this PR though.